draft-ietf-taps-impl.md

```diff
@@ -864,27 +863,25 @@ For example, the mapping of the `Send` function for TCP applies to Connections i
 Each protocol has a notion of Connectedness. Possible values for Connectedness are:
 
 - Connectionless. Connectionless protocols do not establish explicit state between endpoints, and do not perform a handshake during Connection establishment.
-- Connected. Connected protocols establish state between endpoints, and perform a handshake during Connection establishment. The handshake may be 0-RTT to send data or resume a session, but bidirectional traffic is required to confirm connectedness.
+- Connected. Connected (also called "connection-oriented") protocols establish state between endpoints, and perform a handshake during connection establishment. The handshake may be 0-RTT to send data or resume a session, but bidirectional traffic is required to confirm connectedness.
```

Comment:
I added the "connection-oriented" thing because that term is used instead in the other documents, in some places. If we absolutely only want "Connected", then we should change these other occurrences to be consistent.